src/site/content/en/blog/preconnect-and-dns-prefetch/index.md

```diff
@@ -0,0 +1,142 @@
+
+### Streaming media
+
+Another example where you may want to save some time in the connection phase, but not necessarily start retrieving content right away, is when streaming media from a different origin.
+
+Depending on how your page handles the streamed content, you may want to wait until your scripts have loaded and are ready to process the stream. Pre-connecting helps you cut the waiting time to a single round trip once you're ready to start fetching.
+
+## How to implement `rel=preconnect`
+
+One way of initiating a `preconnect` is adding a `<link>` tag to the `<head>` of the document.
+
+```html
+<head>
+    <link rel="preconnect" href="https://example.com">
+</head>
+```
+
+Pre-connecting is only effective for domains other than the origin domain, so you shouldn't use it for your site. 
+
+{% Aside 'caution' %}
+Only pre-connect to critical origins you will use soon because if the connection isn't used within 10 seconds the browser closes it. There's a limit to the number of simultaneous connections a browser can handle (for example, Chrome can handle six), and unnecessary pre-connecting can delay other important resources.
+{% endAside %}
+
+You can also initiate a preconnect via the [`Link` HTTP header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Link):
+
+`Link: <https://example.com/>; rel=preconnect`
+
+{% Aside %}
+A benefit of specifying a preconnect hint in the HTTP header is that it doesn't rely on markup being parsed and it can be triggered by requests for stylesheets, scripts and more. For example, Google Fonts sends a `Link` header in the stylesheet response in order to pre-connect the domain that hosts the font files.
+{% endAside %}
+
+Some types of resources, such as fonts, are loaded in ["anonymous mode"](https://www.w3.org/TR/css-fonts-3/#font-fetching-requirements). For those you must set the `crossorigin` attribute with the `preconnect` hint.
+
+```html
+<link rel="preconnect" href="https://example.com/ComicSans" crossorigin>
+```
+
+If you omit the `crossorigin` attribute, the browser only performs the DNS lookup.
+
+## Resolve domain name early with `rel=dns-prefetch`
+
+You remember sites by their names, but servers remember them by IP addresses. This is why the domain name system (DNS) exists. The browser uses DNS to convert the site name to an IP address. This process — [domain name resolution](https://hacks.mozilla.org/2018/05/a-cartoon-intro-to-dns-over-https/)— is the first step in establishing a connection. 
+
+Because there's a limit to the number of connections a browser can open, you sometimes can't pre-connect to all your important domains. In that case use  `<link rel=dns-prefetch>` to save time on the first step—the DNS lookup which usually takes around [20-120 ms](https://www.keycdn.com/support/reduce-dns-lookups). 
```

Comment:
```suggestion
Because there's a limit to the number of connections a browser can open, you sometimes can't pre-connect to all your important domains. In that case use  `<link rel=dns-prefetch>` to save time on the first step—the DNS lookup, which usually takes around [20–120 ms](https://www.keycdn.com/support/reduce-dns-lookups). 
```
Comma for the nonrestrictive modifier and en dash for the range.